source/docs/software/advanced-controls/introduction/introduction-to-pid.rst

```diff
@@ -157,3 +157,9 @@ An *underdamped* response oscillates around the :term:`reference` before settlin
    :align: center
 
 is slow to rise and does not overshoot the :term:`reference`. A *critically damped* response has the fastest :term:`rise time` without overshooting the :term:`reference`.
+
+Other Information
+------------------
+
+* https://file.tavsys.net/control/controls-engineering-in-frc.pdf - See section 2 and section 6.
```

Comment:
The correct references would be chapter 2 "PID controllers" and section 7.8 "Feedforward"; in `a.b.c.d`, `a` is chapter, `b` is section, `c` is subsection, and `d` is subsubsection.

By the way, this page is already a verbatim copy of chapter 2.